processor/resourcedetectionprocessor/internal/aws/eks/config.go

```diff
@@ -4,15 +4,26 @@
 package eks // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor/internal/aws/eks"
 
 import (
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws/retry"
+
 	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor/internal/aws/eks/internal/metadata"
 )
 
 type Config struct {
 	ResourceAttributes metadata.ResourceAttributesConfig `mapstructure:"resource_attributes"`
+	NodeFromEnvVar     string                            `mapstructure:"node_from_env_var"`
+	IMDSCheckMaxRetry  int                               `mapstructure:"imds_check_max_retry"`
+	MaxAttempts        int                               `mapstructure:"max_attempts"`
+	MaxBackoff         time.Duration                     `mapstructure:"max_backoff"`
```

Comment:
Should these be documented in `README.md`?

Reply:
correct, Readme is not done yet